essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md

```diff
@@ -1,154 +1,208 @@
+
+Once you're ready, start looking for a PR to work on. If you need help getting started, look at AppFlowy's <a href="https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions">Documentation</a>.
+
+### Finding an Issue to Work On
+
+**Step 1:** Go to [AppFlowy's repository](https://github.com/orgs/AppFlowy-IO/repositories) section on GitHub. This is a list of all of AppFlowy's repositories open for contributions. For the purposes of this article, we shall be looking for *Issues* within the main [AppFlowy](https://github.com/AppFlowy-IO/AppFlowy) repository.
+
   ![image](https://github.com/JRS296/AppFlowy-Docs/assets/70965472/715c8a61-af5f-4856-840f-11f8604e765c)
-  *Appflowy Github Page - With all open-source repositories that can be worked on*
-  
-  **Step 2:** Go to the <b>Issues</b> section of the repository. It should look something like this:
-  
+  *AppFlowy Github Page - With all open-source repositories that can be worked on*
+
+**Step 2:** Go to the *Issues* section of the repository. It should look something like this:
+
   ![image](https://github.com/JRS296/AppFlowy-Docs/assets/70965472/14fdc0e3-1517-40da-9576-4646b0516b36)
   *Issues Section*
-  
-  **Step 3:** Go to the label subsection, and type in <b>Good First Issue for Devs</b>.
-  
+
+ **Step 3:** Go to the label subsection, and type in "Good First Issue for Devs"
+
   ![image](https://github.com/JRS296/AppFlowy-Docs/assets/70965472/f5dedcd1-5f08-483d-9898-6372022060b3)
   *Label Subsections Within Issues*
-  
-  **Step 4:** Here, you can go through a list of Issues that can be worked on. They're fairly straightforward and can help you get accustomed to the  development environment as well as the codebase. Feel free to ask anyone from the Appflowy team for suggestions or to assign you to an Issue.
-  
+
+**Step 4:** Here, you can go through a list of *Issues* that can be worked on. They're fairly straightforward and can help you get accustomed to the development environment as well as the codebase. 
+
+Feel free to ask anyone from the AppFlowy team for suggestions or to assign you to an Issue.
+
   ![image](https://github.com/JRS296/AppFlowy-Docs/assets/70965472/e1022103-e98e-42b5-a43d-03f5c5b08f04)
   *Good First Issue for Devs*
-  
-  <p> Issues are generally categorised into either <b>Feature [FR]</b> or <b>Bugs [Bug]</b>. You can take a look at the description provided for the given issue. Send a message down in the comments section to get the issue assigned to yourself. Once that's done, you are ready to begin coding!</p>
-  
+
+Issues are generally categorized into either a <b>Feature [FR]</b> or a <b>Bug [Bug]</b>. You can take a look at the description provided for the given issue. 
+
+Send a message down in the comments section to get the issue assigned to yourself. Once that's done, you are ready to begin coding!
```

Comment:
Maybe add a note saying something about if there is no one assigned to the issue, don't worry about being assigned before starting work on it. Even if you aren't assigned, all pull requests that solve a GitHub issue is appreciated. 

Of course it's best to not work one something someone else is working on, but that's not something we can always control.

Reply:
Maybe we could add caveat that you can work on issues that haven't been assigned to you but there is a greater possibility this will lead to merge conflicts should it end up being assigned to someone else.

Reply:
@JRS296 , would you like to add a note according to @Xazin and @egp415 suggestions?

Reply:
Hello @annieappflowy, got caught up with something else, can I send in the note by tomorrow morning? Thanks.